### ISSUE TITLE
add a set of default dataTypes

### ISSUE THREAD

**Issue by sennierer** [dasch124/modeller#3]
currently every dataType must be explicitly declared. Given that most of the time the same dataTypes are used: `int`, `float`, `str` etc. it would be helpful to directly include these common dataTypes in the model. Maybe [xsd datatypes](https://www.w3.org/TR/xmlschema-2/#datatype) could be reused.

in relation to #1

**Comment by dasch124** [comment]
yes, good point, I'll see how that might be easiest to implement
